src/components/PlayerView/events.ts

```diff
@@ -346,6 +347,10 @@ interface EventProps {
    * Emitted when the current video playback quality has changed.
    */
   onVideoPlaybackQualityChanged: VideoPlaybackQualityChangedEvent;
+  /**
+   * Emitted when a download was finished.
+   */
+  onDownloadFinished: DownloadFinishedEvent;
```

Comment:
Please reorder `onDownloadFinished` to match the current alphabetically ordered event listeners here.

Reply:
Address in 88ef0dfe897b6d62ebd12d5e917bfd4bb454ac2d